src/sidebar.js

```diff
@@ -71,7 +71,15 @@ const initializeRoamJSSidebarFeatures = (extensionAPI) => {
       const rightSidebarTopbar =
         rightSidebar.getElementsByClassName("bp3-icon-menu-open")?.[0]
           ?.parentElement?.parentElement?.firstElementChild;
-      if (rightSidebarTopbar && extensionAPI.settings.get("ws-exp-col")) {
+      if (
+        rightSidebarTopbar &&
+        extensionAPI.settings.get("ws-exp-col") &&
+        !rightSidebarTopbar.hasAttribute("data-roamjs-sidebar-expcolall")
+      ) {
+        rightSidebarTopbar.setAttribute(
+          "data-roamjs-sidebar-expcolall",
+          "true"
+        );
```

Comment:
I wasn't able to reproduce the config issue, but this should at least make it so that multiple icons don't appear

Reply:
Does that mean that the features weren't enabled onload while switched off in config? I'm definitely getting the buttons showing even when the switches are off. 

(I just saw where you refer to the config switches in sidebar.js, sorry for not seeing sooner)

Reply:
That would be unexpected, we check to see if the switch is enabled right there in line 76

`extensionAPI.settings.get("ws-exp-col")`

When the switch is off but the icon is there, are you able to inspect what value this setting has that's returning truthy?